config/tess.example.yml

```diff
@@ -42,6 +42,7 @@ default: &default
     directory_tabs: ['trainers', 'content_providers', 'nodes']
     # The order in with the 'about us' tabs appear
     about_us_tab_order: ['tess_club', 'contact', 'team', 'funding', 'acknowledgements', 'cite']
+    content_provider_sidebar_skip: []
```

Comment:
Does it make sense to rename this `content_provider_disabled` and move under the `features` section, like https://github.com/ElixirTeSS/TeSS/blob/f8349fbb85a16c1a294e2c9f606a95d700a1edb2/config/tess.example.yml#L78

?